src/index.tsx

```diff
@@ -10,14 +10,14 @@ window.MonacoEnvironment = window.MonacoEnvironment || {
   getWorkerUrl(_: string, label: string) {
     // for json worker
     if (label === 'json') {
-      return '../../js/monaco/json.worker.bundle.js';
+      return './js/monaco/json.worker.bundle.js';
     }
     // for yaml worker
     if (label === 'yaml' || label === 'yml') {
-      return '../../js/monaco/yaml.worker.bundle.js';
+      return './js/monaco/yaml.worker.bundle.js';
     }
     // for core editor worker
-    return '../../js/monaco/editor.worker.bundle.js';
+    return './js/monaco/editor.worker.bundle.js';
   },
```

Comment:
I assume the former paths just worked by accident because `..` does nothing when already at the root path.

Reply:
Right, if we load bundled JS file in the `index.html` then we can use path from root. In the preview I see that it works, but there is a question: why does it also work in current deploy on https://studio.asyncapi.com where we have that old path? 🤔

Reply:
Also I wonder if we should use it here `process.env.PUBLIC_URL` to be more safe?